dowhy/causal_model.py

```diff
@@ -459,4 +460,48 @@ def summary(self, print_to_stdout=False):
             print(summary_text)
         return summary_text
 
+    def refute_graph(self, k= 1, method_name =None, test_set = None ):
```

Comment:
`test_set` has a different meaning in ML: the held-out dataset.
How about naming it `independence_constraints`?